test/pylib/scylla_cluster.py

```diff
@@ -32,12 +34,25 @@
 from cassandra.cluster import EXEC_PROFILE_DEFAULT  # pylint: disable=no-name-in-module
 from cassandra.policies import WhiteListRoundRobinPolicy  # type: ignore
 
-#
-# Put all Scylla options in a template file. Sic: if you make a typo in the
-# configuration file, Scylla will boot fine and ignore the setting.
-# Always check the error log after modifying the template.
-#
-SCYLLA_CONF_TEMPLATE = """cluster_name: {cluster_name}
+
+def make_scylla_conf(workdir: pathlib.Path, host_addr: str, seed_addrs: List[str], cluster_name: str) -> dict[str, object]:
+    core = {
+        'cluster_name': cluster_name,
+        'workdir,W': str(workdir.resolve()),
+        'listen_address': host_addr,
+        'rpc_address': host_addr,
+        'api_address': host_addr,
+        'prometheus_address': host_addr,
+        'alternator_address': host_addr,
+        'seed_provider': [{
+            'class_name': 'org.apache.cassandra.locator.SimpleSeedProvider',
+            'parameters': [{
+                'seeds': '{}'.format(','.join(seed_addrs))
+                }]
+            }],
+    }
+
+    return core | yaml.load("""
```

Comment:
You put some of the options in Python syntax above, and some in Yaml syntax below. Isn't it better to have all of them in Python syntax, without this inline yaml string below?

Reply:
Fixed in https://github.com/scylladb/scylladb/pull/11487